crates/bin/pd/src/main.rs

```diff
@@ -441,11 +442,22 @@ async fn main() -> anyhow::Result<()> {
                     (base.join("pd"), Some(base.join("cometbft")))
                 }
             };
-            let genesis_start = pd::migrate::last_block_timestamp(pd_home.clone()).await?;
-            tracing::info!(?genesis_start, "last block timestamp");
             let pd_migrate_span = tracing::error_span!("pd_migrate");
             pd_migrate_span
                 .in_scope(|| tracing::info!("migrating pd state in {}", pd_home.display()));
+
+            if ready_to_start {
+                tracing::info!("disabling halt order in local state");
+                ReadyToStart
+                    .migrate(pd_home, comet_home, None, force)
+                    .instrument(pd_migrate_span)
+                    .await
+                    .context("failed to disable halt bit in local state")?;
+                exit(0)
+            }
+
+            let genesis_start = pd::migrate::last_block_timestamp(pd_home.clone()).await?;
+            tracing::info!(?genesis_start, "last block timestamp");
             Testnet76
                 .migrate(pd_home.clone(), comet_home, Some(genesis_start), force)
```

Comment:
noting that this command *also* applies the testnet76 migration instead of just flipping the ready to start bit. Would it be better to have a separate command that only flips the bit?

Reply:
No, we definitely want to avoid that exact circumstance. As written, if `--ready-to-start` is supplied, then _only_ the halt-bit is modified, and the normal migration is _not_ run. That's because it bails out early via `exit(0)` on L456.

Reply:
I think we can make the case for either, `pd migrate run` vs. `pd migrate ready` or something. Either way, this PR does the job so that we can do the genesis restart with the new halt bit so lgtm